_minutes/README.md

```diff
@@ -10,12 +10,22 @@ After the end of each meeting, meeting notes are published here.
 
 ## Upcoming meetings
 
-- 2022-05-26 at 8 AM PST = https://everytimezone.com/?t=628ec300,3c0
-- 2022-06-09 at 8 AM PST = https://everytimezone.com/?t=62a13800,3c0
+- 2022-07-21 at 8 AM PST = https://everytimezone.com/?t=62d89700,3c0
+- 2022-08-04 at 8 AM PST = https://everytimezone.com/?t=62eb0c00,3c0
 
 ## Past meetings
 
+* 2022-07-07 ([minutes](2022-07-07-wecg.md))
+* 2022-06-23 ([minutes](2022-06-23-wecg.md))
+* 2022-06-09 ([minutes](2022-06-09-wecg.md))
+* 2022-05-26 ([minutes](2022-05-26-wecg.md))
 * 2022-05-12 ([minutes](2022-05-12-wecg.md))
+
+<details>
+<summary>Older meeting notes</summary><br>
```

Comment:
While the current PR preserves the `git blame` history of the meetings that we have had so far, it will force changes on new ones, when the "oldest recent" meeting note gets moved to the section below.

To preserve the `git blame` history, could we rename "Older meeting notes" to something like "All (past) meeting notes", which has a full list of all meeting notes (the current list)? We would basically end up with the list before this PR, plus a volatile copy of the full list above the "archive" (i.e. the list of 5 meeting notes).

Reply:
Ah, I hadn't considered preserving blame. I'm happy to run with this suggestion; I'm tentatively planning to use "All past meeting notes".